src/Discord.Net.Rest/Entities/AuditLogs/DataTypes/ScheduledEventUpdateAuditLogData.cs

```diff
@@ -70,8 +70,7 @@ internal static ScheduledEventUpdateAuditLogData Create(BaseDiscordClient discor
                 status?.NewValue.ToObject<GuildScheduledEventStatus>(discord.ApiClient.Serializer),
                 entityType?.NewValue.ToObject<GuildScheduledEventType>(discord.ApiClient.Serializer),
                 entityId?.NewValue.ToObject<ulong?>(discord.ApiClient.Serializer),
-                entityMetadata?.NewValue.ToObject<GuildScheduledEventEntityMetadata>(discord.ApiClient.Serializer)
-                    ?.Location.GetValueOrDefault(),
+                location == null ? null : location.NewValue?.ToObject<string>(discord.ApiClient.Serializer) ?? Optional<string>.Unspecified,
```

Comment:
Same here

Reply:
you literally cannot convert this in a null coalescing expression because of bracket and calling rules